tests/typings/sync/sync.ts

```diff
@@ -49,6 +49,27 @@ const executeResult = browser.execute(function (x: number) {
 }, 4)
 executeResult.toFixed(2)
 
+const executeAsyncResult = browser.executeAsync(function(a: any, b: any, c: any, d: any, done: any) {
```

Comment:
types should be propagated
```suggestion
const executeAsyncResult = browser.executeAsync(function(a, b, c, d, done) {
```